backend/src/resolvers/wishGroup.ts

```diff
@@ -1,19 +1,20 @@
-import Pool from '../../config/dbConnect';
+import { getWishGroup, editWishGroupColor, editWishGroupTitle, CreateWishGroup, DeleteWishGroup, getWishes } from '../handlers/wishGroupHandler';
```

Comment:
I think we can just import the entire wishgroupHandler file for each resolver file

Reply:
![1598495774964](https://github.com/cameronl10/PadPals/assets/44252100/1b2e71e8-8cb4-4cff-801b-22e6b97d7764)